src/ViewRendererBundle/objects/helper/TPkgViewRendererLessCompiler.php

```diff
@@ -166,7 +166,7 @@ public function getGeneratedCssForPortal($portal, $minifyCss = null)
 
         $lessFiles = $this->getLessFilesFromSnippetsAndTheme($portal);
 
-        if (true === class_exists('Less_Cache')) {
+        if (true === class_exists('Less_Parser')) {
```

Comment:
Since we already checked if `Less_Parser` exists in line 157, we could skip the check hier. 

If there is reason to believe, that `Less_Cache` does not exist, even though `Less_Parser` does, then we should add the check to line 196.
`if (_DEVELOPMENT_MODE && true === class_exists('Less_Cache')) {`

Reply:
That is actually an old "if-else": if Less_Parser does not exist then "lessc" is used (last line calls getCssFromLessC()).